src/App/Service/Github/Filters.php

```diff
@@ -9,6 +9,7 @@ class Filters
 
     public const FILTER_AUTHOR = 'author';
     public const FILTER_FILE_EXTENSION = 'fileExtension';
+    public const FILTER_MILESTONE = 'milestone';
```

Comment:
I dont see where this would be used

Reply:
ho yes ! a refactor later it's not used anymore. Thx !